deploy/metrics-apiservice.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.metrics
+spec:
+  service:
+    name: metrics-server
+    namespace: kube-system
+  group: metrics
```

Comment:
Again, why not move to `metrics.k8s.io` now when we can?

Reply:
Will do once https://github.com/kubernetes/kubernetes/pull/51653 is merged.